test/models/gpg_keys_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'katello_test_helper'
+
+module Katello
+  class GpgKeyTest < ActiveSupport::TestCase
+    should allow_values(*valid_name_list).for(:name)
+    should_not allow_values(*invalid_name_list).for(:name)
+    should_not allow_values(['', ' ', "\t", *RFauxFactory.gen_strings(247).values]).for(:content)
```

Comment:
this should be rewritten to (as the current implementation will be interpreted as one  array value, and will not be allowed any way):
```ruby
should_not allow_values('', ' ', "\t", *RFauxFactory.gen_strings(247).values).for(:content)
```